api/swagger.yaml

```diff
@@ -33,6 +33,21 @@ definitions:
         x-go-name: NumOfDeployedVersions
     type: object
     x-go-package: github.com/aerogear/mobile-security-service/pkg/models
+  AppInitRes:
```

Comment:
@davidffrench it is an extra change of this PR. I just added in order to help you with the /init